### DISCUSSION TITLE
Notebook on Stereoisomer and Tautomer generation

### DISCUSSION THREAD

**Discussion by sitanshubhunia** [PatWalters/practical_cheminformatics_tutorials#9] [General]
Thanks for your helpful resources @PatWalters . It's helped me a lot to learn. Requesting you to make a useful notebook on how to generate  **Stereoisomer** and **Tautomer** . That would be very helpful to me.

**Comment by PatWalters** [comment]
I like that idea, thanks!  I'll work on it.

**Comment by PatWalters** [comment]
I added a new tutorial, "Enumerating Stereoisomers and Tautomers".   Please let me know if this covers the material you're looking for. 
https://colab.research.google.com/github/PatWalters/practical_cheminformatics_tutorials/blob/main/fundamentals/stereo_and_tautomers.ipynb
